.github/workflows/validate.yml

```diff
@@ -56,19 +56,16 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Checkout GitHub merge
```

Comment:
There doesn't seem to be any benefit of this, actions/checkout@v2 uses merge commit by default.